### DISCUSSION TITLE
Where php.ini for CLI is located?

### DISCUSSION THREAD

**Discussion by kossa** [beyondcode/herd-community#1146] [Q&A] [answered]
Hello,

I want to change some setting ini php.ini for CLI, where can I locate?

<img width="1002" alt="image" src="https://github.com/user-attachments/assets/f9294f29-2d26-4e88-9841-846b11a40894">

<img width="1162" alt="image" src="https://github.com/user-attachments/assets/c050c2d7-55f1-4265-8092-45119b11d72e">

**Comment by mpociot** [comment, marked as answer]
The ini file in your screenshot is the one you're looking for 😅
You can also use the "herd ini" command to open the ini file in your configured IDE.

**Reply by kossa** [reply]
Ok bcz on the server there is 2 one for FPM and another for CLI, thank you

**Comment by 13garth** [comment]
I am stuck here with herd PHP ini. 
I found the php.ini files. But there is practically nothing inside of them... 

all my php.ini files look like this with just 4 lines. 

```
curl.cainfo=/Users/username/Library/Application Support/Herd/config/php/cacert.pem
openssl.cafile=/Users/username/Library/Application Support/Herd/config/php/cacert.pem
pcre.jit=0
output_buffering=4096
```

How can I enable intl for my php?
